release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md

```diff
@@ -0,0 +1,6 @@
+# .NET Core 3.0 API Changes
+
+The following API changes were made in .NET Core 3.0:
+
+- [.NET Core](./3.0.0.md)
+- [.NET Core NuGet Packages](./standalone-packages/3.0.0-standalone-packages.md)
```

Comment:
Do we want to add aspnet?

Reply:
They didn't publish any ref assemblies for 2.1 and 2.2. So I am trying to find a way to get the diff but it seems unlikely

Reply:
Sounds good to me. Thanks for checking.